app/models/profile.rb

```diff
@@ -61,6 +61,12 @@ def canonical?
     parent_profile_id.blank?
   end
 
+  def policy
+    Profile.includes(:benchmark)
```

Comment:
This can probably return safely just `self` if external is false, isn't it? There can't be more policies with the same ref_id/account_id/benchmark_ref_id in the same account

Reply:
Yep! Good point.